app/locales/en/translations.js

```diff
@@ -6,6 +6,7 @@ export default {
   "language.en": "English",
   "language.zh": "中文",
   switch_language: "Switch language",
+  add_inventory_item: "Add item to inventory",
```

Comment:
@shreyas098 did you forget to add chinese translation ?

Reply:
I have asked for it in volunteers group but didn't get any response yet, so will create separate  ticket and add it later once i get the translation @abhinav098 .

Reply:
@shreyas098 , Please add the same translation in chinese file as well, because when the app will be in chinese mode, this key won't be found and it will say translations missing. 

You can update it later once you get the chinese translation for this.

cc- @namrataukirde

Reply:
Okay